silvestre-ffi/tests/smoke_test.c

```
@@ -0,0 +1,59 @@
+    assert(err != NULL);
+    printf("expected error: %s\n", err);
+
+    /* Create a 1x1 red RGBA image from buffer. */
+    uint8_t pixel[4] = {255, 0, 0, 255};
+    img = silvestre_image_from_buffer(pixel, 4, 1, 1);
+    assert(img != NULL);
+    assert(silvestre_image_width(img) == 1);
+    assert(silvestre_image_height(img) == 1);
+    assert(silvestre_image_pixels_len(img) == 4);
+
+    const uint8_t *px = silvestre_image_pixels(img);
```

Clang (14.0.6)

[warning] variable 'px' is not initialized

(cppcoreguidelines-init-variables)